ModelVale/ModelVale/CustomComponents/HealthBarView.m

```diff
@@ -0,0 +1,68 @@
+//
+//  HealthBar.m
+//  ModelVale
+//
+//  Created by Chaytan Inman on 7/14/22.
+//
+
+#import "HealthBarView.h"
+
+@interface HealthBarView()
+@property (nonatomic, strong) CAShapeLayer* shapeLayer;
+@property (nonatomic, assign) NSInteger width;
+@property (nonatomic, assign) CGPoint leftTopPoint;
+@property (nonatomic, assign) CGPoint rightTopPoint;
+@property (nonatomic, assign) CGPoint leftBottomPoint;
+@property (nonatomic, assign) CGPoint rightBottomPoint;
+@property (nonatomic, assign) CGPoint topMidPoint;
+@property (nonatomic, assign) CGPoint bottomMidPoint;
+
+@end
+
+@implementation HealthBarView
+
+
+
+// Only override drawRect: if you perform custom drawing.
+// An empty implementation adversely affects performance during animation.
+- (void)drawRect:(CGRect)rect {
+    [super drawRect:rect];
+    [self initPoints];
+    self.shapeLayer = [CAShapeLayer new];
+    self.shapeLayer.fillColor = UIColor.blueColor.CGColor;
+    self.shapeLayer.frame = CGRectMake(0, 0, 500, 500);
+    
+    [self drawLines:self.leftTopPoint to:self.rightTopPoint bendFactor:0.5 thickness:10.0];
+//    [self.shapeLayer.frame ]
+    [self.layer addSublayer:self.shapeLayer];
+}
+
+- (void) initPoints {
+    // Top left is 0,0
+    NSInteger margin = 10; // todo calc margin based on width
```

Comment:
It is very good that you are adding TODOs just like in your comments before, but try to standardized how you are going to write this comments so it is easier for you to find this TODOs